[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.68:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.97:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.97:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.97:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.97:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.68:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:02:13Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active    2/3  opensearch                                 0  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
opensearch/0*                active    idle   1        10.222.180.68   9200/tcp  
opensearch/1                 active    idle   2        10.222.180.206  9200/tcp  
opensearch/2                 unknown   lost   3        10.222.180.97   9200/tcp  agent lost, see 'juju show-status-log opensearch/2'
self-signed-certificates/0*  active    idle   0        10.222.180.226            

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.222.180.226  juju-6422d1-0  ubuntu@22.04      Running
1        started  10.222.180.68   juju-6422d1-1  ubuntu@22.04      Running
2        started  10.222.180.206  juju-6422d1-2  ubuntu@22.04      Running
3        down     10.222.180.97   juju-6422d1-3  ubuntu@22.04      Running

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:02:13 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:02:08)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:01:24)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [lost (since: 01:57:21)] unknown: agent lost, see 'juju show-status-log opensearch/2'

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:02:43 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:02:08)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:01:24)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [lost (since: 01:57:21)] unknown: agent lost, see 'juju show-status-log opensearch/2'

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:03:10 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [executing (since: 02:03:00)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [executing (since: 02:03:03)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:00)] maintenance: Requesting new TLS certificates...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:03:22 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [executing (since: 02:03:21)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:03:15)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:00)] waiting: The OpenSearch service stopped.

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:03:33 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:03:22)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:03:15)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:00)] waiting: Waiting for OpenSearch to start...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:03:44 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:03:22)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:03:15)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:00)] waiting: Waiting for OpenSearch to start...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:03:56 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [executing (since: 02:03:50)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [executing (since: 02:03:50)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:50)] waiting: Waiting for OpenSearch to start...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:04:08 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:03)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:02)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:03:50)] waiting: Waiting for OpenSearch to start...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:04:19 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [executing (since: 02:04:18)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [executing (since: 02:04:19)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:04:31 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:04:43 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:04:54 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:05:06 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:05:17 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:05:29 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.615* -- (10.222.180.68) -- [idle (since: 02:04:20)] active: 
		opensearch-1.615  -- (10.222.180.206) -- [idle (since: 02:04:19)] active: 
		opensearch-2.615  -- (10.222.180.97) -- [idle (since: 02:04:23)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:05:40 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 02:05:42 -- Waiting for model: complete.



[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[1m[31mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.197:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.68:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.197:9200/_nodes
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=839, count=840, last_expected_id=839)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.222.180.68:9200/series_index/_search_shards
[32mINFO    [0m integration.ha.conftest:conftest.py:48 



The writes have been cleared.